pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "appoinment-scheduler"
+version = "0.1.0"
+description = "Appoinment Scheduller API PET project"
+authors = ["Vasyl Boliuk <vbboliuk@gmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.12"
+ruff = "^0.5.5"
+bandit = "^1.7.9"
+commitizen = "^3.28.0"
+git-changelog = "^2.5.2"
+pre-commit = "^3.7.1"
```

Comment:
@vasylboliuk please move ruff, bandit, commitizen, git-changelog and pre-commit dependencies under `dev` section since this section will be used by Docker for our docker image and should be as minimal as possible

Reply:
Done

Reply:
@vasylboliuk please move all dependencies that are not directly related to the functionality of the application under the poetry dev group (git-changelog, pre-commit, bandit, commitizen